RIOT-OS/RIOT
# makefiles: allow setting PAN ID like channel

### Contribution description

The default PAN ID configuration should be just as easy to set as the default channel.

This adds an option for this to `default-channel.ink.mk`

### Testing procedure

You should be able to easiely set the PAN ID on the command line:

```
    make DEFAULT_PAN_ID=0xabcd flash
```

### Issues/PRs references
none
